ui/src/adapters/api/connections.ts

```diff
@@ -52,16 +57,18 @@ export async function getConnection({
 export async function getConnections({
   credentials,
   env,
-  params,
+  params: { max_results, page, query },
```

Comment:
We should apply also here what Elias mentioned in [this comment](https://github.com/0xPolygonID/issuer-node/pull/564#discussion_r1431197960).